src/components/Profile/ProfileHeader/index.js

```diff
@@ -1,12 +1,18 @@
 import React, { memo } from 'react'
 import './styles.scss'
+import {
+  getMe,
+  getProfiles as getProfilesAction,
+  getProfilesInChannel as getProfilesInChannelAction,
+} from 'mattermost-redux/actions/users'
 
 const Header = props => {
-  const { nickname, location } = props || ''
+  const { nickname, location, startDirect, currentUser } = props || ''
 
   return (
     <span className="profile-header-item">
       <h1>{nickname}</h1>
+      {!currentUser && <button onClick={startDirect}>Aloita keskustelu</button>}
```

Comment:
There is a ButtonContainer, which already has tabIndex and other keyboard events implemented. Consider using that insted of `<button>`